packages/libc/libc.0.0.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Raw definitions and bindings to platforms system libraries"
+description:
+  "`libc provides definitions (and in the future bindings) to easily interoperate with C code (or C-like code) on several platforms, and it does this with a target and architecture aware API."
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+tags: ["libc" "platform" "bindings" "raw bindings"]
+homepage: "https://github.com/ocaml-sys/libc.ml"
+bug-reports: "https://github.com/ocaml-sys/libc.ml/issues"
+depends: [
+  "config" {>= "0.0.1"}
+  "ocaml" {>= "5.1.1"}
+  "dune" {>= "3.12"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocaml-sys/libc.ml.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/ocaml-sys/libc.ml.git"
available: arch != "x86_32" & arch != "arm32" & arch != "ppc64"
```
Due to 
```
#=== ERROR while compiling libc.0.0.1 =========================================#
# context              2.2.0~beta2~dev | linux/x86_32 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/ocaml-sys/libc.ml/releases/download/0.0.1/libc-0.0.1.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/libc.0.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p libc -j 71 @install
# exit-code            1
# env-file             ~/.opam/log/libc-7-69729d.env
# output-file          ~/.opam/log/libc-7-69729d.out
### output ###
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc -w -40 -g -bin-annot -I libc/.libc.objs/byte -no-alias-deps -open Libc__ -o libc/.libc.objs/byte/libc.cmo -c -impl libc/libc.pp.ml)
# File "libc/libc.ml", line 19, characters 8-12:
# 19 | include Impl
#              ^^^^
# Error: Unbound module Impl
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc -w -40 -g -bin-annot -I libc/.libc.objs/byte -no-alias-deps -open Libc__ -o libc/.libc.objs/byte/libc__Linux_like.cmo -c -impl libc/linux_like.pp.ml)
# File "libc/linux_like.ml", line 14, characters 23-33:
# 14 | let rand_max : c_int = 2147483647
#                             ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc -w -40 -g -bin-annot -I libc/.libc.objs/byte -no-alias-deps -open Libc__ -o libc/.libc.objs/byte/libc__Libc_aarch64_apple_darwin.cmo -c -impl libc/libc_aarch64_apple_darwin.pp.ml)
# File "libc/libc_aarch64_apple_darwin.ml", line 94, characters 23-33:
# 94 | let rand_max : c_int = 2147483647
#                             ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```
and
```
#=== ERROR while compiling libc.0.0.1 =========================================#
# context              2.2.0~beta2~dev | linux/ppc64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/ocaml-sys/libc.ml/releases/download/0.0.1/libc-0.0.1.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/libc.0.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p libc -j 175 @install
# exit-code            1
# env-file             ~/.opam/log/libc-7-b622aa.env
# output-file          ~/.opam/log/libc-7-b622aa.out
### output ###
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc -w -40 -g -bin-annot -I libc/.libc.objs/byte -no-alias-deps -open Libc__ -o libc/.libc.objs/byte/libc.cmo -c -impl libc/libc.pp.ml)
# File "libc/libc.ml", line 19, characters 8-12:
# 19 | include Impl
#              ^^^^
# Error: Unbound module Impl
```

Other failures seem CI related, I am rerunning the jobs

Reply:
How can I add these to my dune-project?

Reply:
See https://github.com/ocaml-sys/libc.ml/pull/3

Reply:
Thank you!